src/Component/Document.php

```diff
@@ -27,6 +32,7 @@ public function __construct(string $link)
         }
 
         $this->link = $link;
+        $this->filename = empty($filename) ? 'document' : $filename;
```

Comment:
Is it necessary to provide a default file name?

Reply:
Not totally, but in my tests, unamed files are send as "null.pdf", what made the file thumb not load in the whatsapp conversation. so call it "document.pdf" by default fixed this little problem (from the whatsapp api)